### DISCUSSION TITLE
Issue with xRDP and Linux Mint Xfce edition desktop

### DISCUSSION THREAD

**Discussion by sqwuade** [neutrinolabs/xrdp#2758] [Q&A]
Greetings,
From researching this issue I've seen people reporting the same basic problem going back to at lease 2014...
I'm not sure if it's an issue with Linux Mint Xfce edition or with xRDP but when you install Mint Xfce edition and login to a remote session with xRDP it completely rearranges and changes the desktop environment.  When you login locally to the machine again you'll find that everything has been changed around locally too.
Has anyone worked on this issue from a dev perspective?  Here's one example of someone reporting the issue:

https://forums.linuxmint.com/viewtopic.php?t=364069

I run LinuxLite OS too, which uses a modified version of Xfce as it's desktop, and have not seen the issue on that OS.

Thanks for any info

**Comment by matt335672** [comment]
Hi @sqwuade 

I suspect you're using the same user to log on on the console and connect over xrdp.

If so, this isn't supported on systemd-based systems. This is explained further [in the FAQ](https://github.com/neutrinolabs/xrdp/wiki/Tips-and-FAQ#why-cant-i-log-the-same-user-on-on-the-graphical-console-and-over-xrdp-at-the-same-time). The two sessions will end up wrapped around each other. Use a different user for the xrdp connection.

**Reply by sqwuade** [reply]
Thanks for the suggestion Matt.  I'll give that a try in the next couple of days.  It's not that I can't login to a session it's that logging in over xRDP messes up the desktop settings on the target machine.  It only does it on the Mint Xfce version.  If I use a Cinnamon session it works fine .

**Reply by matt335672** [reply]
Please feed back what you find @sqwuade. There may be something else going on that I've not anticipated.
